components/dialog/overview.md

```diff
@@ -45,38 +47,108 @@ The following example demonstrates how to set up the Dialog with its default con
 }
 ````
 
->caption The result from the above code snippet.
+## Predefined Dialogs
+
+Predefined Dialogs are styled substitutes to the standard confirm, alert and prompt dialogs. They are separate from the main Dialog component. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
```

Comment:
What do you want to say with "They are separate from the main Dialog component." - what should the reader do or learn from this information?

```suggestion
Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. They are separate from the main Dialog component. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
```